pkg/sql/opt/exec/execbuilder/tests/5node/5node_test: TestExecBuild_explain_analyze_plans failed

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_explain_analyze_plans [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21066895?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21066895?buildTab=artifacts#/) on master @ [25fa8b9ecb40122caceaf12e2874c6e15e4e95ca](https://github.com/cockroachdb/cockroach/commits/25fa8b9ecb40122caceaf12e2874c6e15e4e95ca):


```
            regions: <hidden>
            ·
            • group (streaming)
            │ sql nodes: <hidden>
            │ regions: <hidden>
            │ execution time: 0µs
            │ actual row count: 5
            │ group by: k
            │ ordered: +k
            │
            └── • merge join
                │ sql nodes: <hidden>
                │ regions: <hidden>
                │ execution time: 0µs
                │ actual row count: 5
                │ equality: (k) = (k)
                │ left cols are key
                │ right cols are key
                │
                ├── • scan
                │     sql nodes: <hidden>
                │     kv nodes: <hidden>
                │     regions: <hidden>
                │     KV time: 0µs
                │     KV rows decoded: 5
                │     actual row count: 5
                │     missing stats
                │     table: kv@kv_pkey
                │     spans: FULL SCAN
                │
                └── • scan
                      sql nodes: <hidden>
                      kv nodes: <hidden>
                      regions: <hidden>
                      KV time: 0µs
                      KV rows decoded: 5
                      actual row count: 5
                      missing stats
                      table: kw@kw_pkey
                      spans: FULL SCAN
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsm29v2k4Sx5_fq1jto1Q_U7z-A8RSJdq0d0qvgSp_KlWnKHLwlFgYm9pLSC7Ky7o3cK_sJ0NosDc2DBDbZOmDCtuL8XxmPLP73ckDjX571KJnX759OTong9v3A4XYt_2DweT94B3552n3hAxuydfucYcMJqTbmQ4hH0h8nfzrtHvxnXz6OT1JFeoHDnTsIUTU-g9lVKEaVahOFWpQhZr0UqGjMOhBFAVhPORh-oVj545aqkJdfzTm8elLhfaCEKj1QLnLPaAWPbevPTgF24GwrlKFOsBt15v-zOC2Pbi9Gg3gnir0KPDGQz-ySPw0ZyM7_lirx0_x7x8k_rHIIj6bHYbQd4N4AIeIz05xdwgWUf__v-hpSDCJiAO9wAHHIk_fu77nEJEQbMciLfJpdrJ_-v2I9GzPi_6MG9luOB8XP8HJj6MjEnEYkV4w9jk5gDted33-ziLq1KjZAIBB1oChfUeGMAzCe2J7XtCzefxY6vQZrm3eu4GIBGM-GnOLxOOnjz8_wejlo0JnRzP4c7jX9-TGjm6SWNvx-EuFRtzuA7XYguuOP1NLfVTW814j5b1JezDJ816d7f23Bf9pKf81Mv33fN8gdCAEJ33fv-IHWWnUC6FwAmEfvgauD2Gdpd5kD37xgzb7692H0O3fzD5ShXZj69tMacdOgDvojbkb-Elf55GNr0W_PcJhOCKOGw3IOLL7sDb4Z6h6CipDvRUf-_0Q-jYPwjozBecptDsDG9tOFfqx8_Oq0z2_6lx8-3bQZjGYs4uTg7YWfzrqXnTOnz6vQeiVY89IYzI3C758lpq6Gcuzi5Or45imHh-dgu9AOI0_0tbqbb0gws_0zBQ9LTvInnmM_Zf4vYiuE9SCUV1LRWCGjUvtMDPtaKTtSEYBW72EMOQEoK7V6vq-hGx7CsDW9V8TOwXY--81pgDNTP8VOAVgOzEF0FefAqDeisUpQKPaU4A0gg2nAI3Ngm_JFIDt4hQgO8iEKUAyyLTVU6-GLZ16rW7sU--2S6e2rv9a2NK5999rlM5Wpv8KLJ3aTpROdfXSiXorFktns9qlM41gw9LZ3Cz4lpRObRdLZ3aQCaUzGWT66qlXx5ZOo1Y396l326VTX9d_h9jSufffa5TOw0z_FVg69Z0onQjhGfVWLJbOVrVL55aF59ZmwbekdOq7WDoRwnMyyIzVU6-BLZ1mbZ94t104jXW9ZyILZ63OiO07hJGA30C49-S2S6iZ6ckCS6ixEyUUsfpEvR-LJfSw2iV0y6vPw82Cb0kJNXaxhCJWnwamQeAUolHgR5Daw335p9TUT9VYzBqcPsx8EwXjsAffw6A3HTs77E5vNK0VDkR8dlWbHRz780sRt3nepvArv8DxA4RgD_-0Ha1uSivTFB_4JAgHxLM5-L37P7bMz09slyetdCCC0LU997-2iGD-temzh9AD93ZeteaX5qVrfm1WvuZXhxDFXBYHqCtWlkU-DAmI6SUTUnMIqVshlIogDUuo7BgqnJCOJKRJF0OGQIghMqpWqYzawJnSyjTlrWZUJKBURi2eUPEZFUuo7BgqPqMiCWnSxZAhENLShNRFQklA6uKtTCE567mwjSpNd1PJxzRxtjBVtlLcQHqbmdIRQsaQJl0MNZExpEkXQ00hhozc9PwCoPmtWgJsM_dWWkNM9ZWZPAu2NNZUVt5o3DAcn-yldvmuZkhbJFRWsITKVlYKX0vqSEISKisCoeaaysobBdTE8clealcgoyJtkVBZwRIqW1kpPqMiCUmorAiEWrnz7cMcZUVIzodrKitvlLUpqFj5gHLkmnL-yC2hGyC9LaGygnS3hMoKMoYkVFbErQFht3WZtFIoIT2HkGYsQ8TWSEQtIYgYZrtVgnWlhgRU9sK7eB0KSyhHvSl_4aShrZHO4ToWkYRSi4hI2HOtlNZSeFptIQGVvfouPq1iCeVIOBVIq2hrpHO4jkUkod4iIhJ2kxMTembkKC5ijhY2XiWXXAR5awkhCVtksEGUo-SUn6UbWIdLqLpgHS6h7CJGUX4binSEWmIQ7ZtbkpMhJKCyF-HFt85jCUnYM4NGVOUeKR1rjYSqi4ho3-GSIHSIBFT2Irz4tIolJGHjDBpRlRuldKw1EqouIqL8NheW1-ci5uh9o0tSdRFFrrU7Xd4oogY2iCTsn8FGUY6UU_7f6jWxDpdQdREcrlW72aX4LUchiLRq97oUX-mRgORTXbCEJFRd0IjKjqLixU0sohwpp_wVlCFaU-1el-Jn9EhA8qkuWEISqi5oRGVHUfFpFYsoR8qpQFoVrcnvddHyel3EHL3vdUnmaEHkWkJIQtUFG0QSqi7YKJKwgQYbRTlSTvlZuik6fN_rklRdjMdLhf7ygsmV61CLqk__ai_8N_9H4y_Y_YhaD_TsJphMaZ3fjyCi1i_bi0ChJ_YAPgOHcOj6bsTdHrV4OIbHx3_8HQAA__-LfhCW
            
[11:17:30] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18138/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans with config 5node: 12 tests, 1 failures
    logic.go:4653: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18138/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans:172: error while processing
    logic.go:4653: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18138/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans:172: too many errors encountered, skipping the rest of the input
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/983e0f95e5f92eec8be065c3c603ddf8/logTestExecBuild_explain_analyze_plans3014832614
--- FAIL: TestExecBuild_explain_analyze_plans (3.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_explain_analyze_plans.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-59519

--- a/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans
+++ b/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans
@@ -116,59 +116,6 @@ regions: <hidden>
 ·
 Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsm-1u2swSx7-fq1jtp1Q1hfULEEuVaNOeo_Q0UOWlUnUURQ6eEgtjU-8SkhPlsp4beK7skU1IsDeYDCE2Yp0PFdhr4_nNeGb3v9M7yv_41KYnX79_PTglw-sPQ40414O94fTD8B3593HviAyvybfeYZcMp6TXTYaQjyQ-T_5z3Dv7QT7_Sg5SjQahC11nBJza_6OMalSnGjWoRk2qUYuea3QchX3gPIziIXfJBYfuDbUbGvWC8UTEh8812g8joPYdFZ7wgdr01Ln04RgcF6J6g2rUBeF4fvIzw-vO8PpiPIRbqtGD0J-MAm6T-GlOxk78sVaPn-K_P0n8Y9wmAZt9jWDghfEAAVzMDglvBDZp_P0XfxgSTjlxoR-64Nrk4brLWwGcROC4NmmTz7ODg-MfB6Tv-D5_HDd2vGg-Ln6Co58HB4QLGJN-OAkE2YMbUfcC8c4mjcSo2QCA4bIBI-eGjGAURrfE8f2w74j4sRrJM1w6on8FnIQTMZ4Im8Tjk8efH2D0_F6js28z-HO4l7fkyuFXaaydePy5RrlwBkBttuC6wy_Ubtxr63mvmfHetDOc5nmvzir_bcB_esZ_zaX-e7pvGLkQgZu97_v4QV406plQOIJoAN9CL4CozjJvsg-_xV6HvX_3MfIGV7OPVKO92PoO0zqxE-AG-hPhhUHa13lk43P8j08EjMbE9fiQTLgzgLXBP0E1MlAZ6q34NBhEMHBEGNWZJTlPo70Z2Nh2qtFP3V8X3d7pRffs-_e9DovBnJwd7XX0-NNB76x7-vB5DUJvHHtmFpP1uuDLZ6k3Xsfy5Ozo4jCmacTfjiFwIUrij3T0escoiPATPStDT18eZE88JsFz_J5F1w1r4biuZyJwiY0r7bCW2tHM2pGOAvbyEsKQE4C6XqsbVQnZ9BSAreu_FnYKUPnvLaYAraX-K3AKwHZtCoB6KxanAE2lpgDN1wXfiikA2_EpQDrI9JenXh1bOo1a3axS76ZLp76u_9rY0ln57y1KZ3up_wosnfqulU7UW7FYOltKlc7W64JvRenUd7x0poPMeHnqNbCl06zVrSr1brp0Guv6bx9bOiv_vUXp3F_qvwJLp7FrpRP1ViyWzrZSpbP9uuBbUTqNHS-d6SAzX556TWzptGpV4t104TTX9Z6FLJy1OiNO4BJGQnEFUeXJTZdQa6knCyyh5q6VUNT7sVhC95UqofuvC74VJdTc8RJqYhoEjoGPw4BDZg_3-Z9qZH6qxmLW4A5g5hseTqI-_IjCfjJ29rWX3CipFS5wMTurz74cBvNTXDgib1P4LV_gBk1MjMAZPbYdvdyU9lJTAhDTMBoS3xEQ9G8fbZkfnzqeSFvpAofIc3zv_46MYH5Z8uwR9MG7ThgsnJqXrsdzCYD52RHwmEv2YiwfhgTEjJIJsRxC7Y0QYmlCOpaQcjFkIAnpZcdQ4YRMiRBDZFR9qzJqE2dKe6kpO-prhgSUyajFEyo-o2IJKRdDBpKQXnYMFZ9RJUJ6llBjkVAaUGPxVpaUnI1c2OY2TXczr5Zl4WxhDdVKcRPpbWYpRwgZQ7pyMdRCxpCuXAy1pBgyc9PzM4Dmt2pLsK3cW-lNOdVvzeRZsqW5prKyo3HDcHyWL7XLdzVD2lK6slK4s3UsIfWUFSSh0pWVwteSpkSotaaysqMh1MLxWb7U3oKMirSldGWl-IyKJaSesoIkVLqyUnxGlQi1c-fb-znKipSc99dUVnY0Gi1JxcoHlCPXlK89NZHeVlBZQbpbQWUFGUMKKivy1oC027pKWtntAtaWYohhdlsViCEdCUi5dTfDEsoRb8pfN-loa8p2eOE5w8AiKl1pKX7HVUYkbbmqLbW0kYCUW3wzLKEcBWcL0iramrIdXnxaxSIqXW4pPq3KiKTN5NR8npk5gouco6V9V8UVF0ndWkFIwQ4ZbBDlCDlboEphHa6g6IJ1uIKqixxF-V0oZRMqXnWRg6jqbUlPhpCAlFuEMywhBVtm0Ii2uUXKwFqjoOoiI6oaXFKE9pGAlFuEMywhBftm0Ii2uU_KwFqjoOoiI8rvcmF5bS5yjq76XNKqiyxyrd3osqOImtggUrB9BhtFOVJO-cJUC-twBVUXyeF61euSVl2kINK3u9el-HZWJKCyVZfiN2WxhBRUXdCIlIsiA4soR8opfwVlytZsd69L4WlV-v-9KwCVrboUn1axhBRUXdCIlIsiA4soR8rZgrQqW5Pf66Ln9brIObrqdUnnaEnkWkFIQdUFG0QKqi7YKFKwgQYbRTlSzhYIU7LDq16XtOpi3p9r9LcfTi88l9q08fBXe-af-R-NL3AGnNp39OQqnCa0Tm_HwKn92_E5aPTIGcIXEBCNvMDjwutTW0QTuL__1z8BAAD__6GIEHc=
 
-skipif littleendian
-query T
-EXPLAIN ANALYZE (DISTSQL) SELECT kv.k, avg(kw.k) FROM kv JOIN kw ON kv.k=kw.k GROUP BY kv.k
-----
-planning time: 10µs
-execution time: 100µs
-distribution: <hidden>
-plan type: custom
-rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
-DistSQL network usage: <hidden>
-regions: <hidden>
-·
-• group (streaming)
-│ sql nodes: <hidden>
-│ regions: <hidden>
-│ execution time: 0µs
-│ actual row count: 5
-│ group by: k
-│ ordered: +k
-│
-└── • merge join
-    │ sql nodes: <hidden>
-    │ regions: <hidden>
-    │ execution time: 0µs
-    │ actual row count: 5
-    │ equality: (k) = (k)
-    │ left cols are key
-    │ right cols are key
-    │
-    ├── • scan
-    │     sql nodes: <hidden>
-    │     kv nodes: <hidden>
-    │     regions: <hidden>
-    │     KV time: 0µs
-    │     KV rows decoded: 5
-    │     actual row count: 5
-    │     missing stats
-    │     table: kv@kv_pkey
-    │     spans: FULL SCAN
-    │
-    └── • scan
-          sql nodes: <hidden>
-          kv nodes: <hidden>
-          regions: <hidden>
-          KV time: 0µs
-          KV rows decoded: 5
-          actual row count: 5
-          missing stats
-          table: kw@kw_pkey
-          spans: FULL SCAN
-·
-Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsm-1u2kgXx78_VzGaT6lqiscvQCxVok37rNJtoMpLpWoVRQ4-JRbGpvYQko1yWXsDe2UrQ2iwJzYcILaToR8qwAP4_M7x-TN_n9zR6JdHLXry-evng1MyuH43UIh93d8bTN4N3pD_H3ePyOCafOkedshgQrqd6RLynsTHyR_H3bNv5OOP6YtUoX7gQMceQkStvyijCtWoQnWqUIMq1KTnCh2FQQ-iKAjjJXfTNxw6N9RSFer6ozGPXz5XaC8IgVp3lLvcA2rRU_vSg2OwHQjrKlWoA9x2venXDK7bg-uL0QBuqUIPAm889COLxGdzMrLjh7V6fBZ_fifxl0UW8dnsaQh9N4gXcIj47CXuDsEi6r__RA9LgklEHOgFDjgWeXjf5S2HiIRgOxZpkY-zF_vH3w5Iz_a86Pe6ke2G83XxGRx9PzggEYcR6QVjn5M9uOF11-dvLKJOg5otABhkLRjaN2QIwyC8JbbnBT2bx6elTs_h0ua9K4hIMOajMbdIvH56-vMXGD2_V-js2Qz-HO7lLbmyo6sk1na8_lyhEbf7QC22kLrDT9RS75X1stdIZW_SHkzysldnu_xtIX9aKn-NzPw9fm4QOhCCk_7ct_GJrLTqiVI4grAPXwLXh7DOUleyBz_5Xpu9ffM-dPtXs4dUod04-jZT2nES4AZ6Y-4GfjLXeWTjY9Evj3AYjojjRgMyjuw-rA3-EaqegspQV8WHfj-Evs2DsM5MIXkK7c7AxrFThX7o_LjodE8vOmdfv-61WQzm5Oxor63Fjw66Z53Th8cZhJ65vow0CnOzAsvnpamb8To5O7o4jInp8bNj8B0IpzVG2lq9rW-R4iMhM0VIyy6Wx5jH_lOMnsTTCWrBqK6lKmndOMzMOBrpOJKZZqtLAUMKeV2r1fWdFGxbytm6-WtipXyXv-eQ8mZm_gqUcvYipFxfXcpRV8WilDfKl_J0mBtKeWOzAlsi5ayqUp5dLIKUJ4tFW72FalgJ1Gt1Y9dCty2B2rr5a2ElcJe_55DAVmb-CpRA7UVIoLq6BKKuikUJbJYvgekwN5TA5mYFtkQCtapKYHaxCBKYLBZ99RaqYyXQqNXNXQvdtgTq6-ZvHyuBu_w9hwTuZ-avQAnUX4QEIgxd1FWxKIGt8iVwy4Zua7MCWyKBelUlEGHoJovFWL2FGlgJNGu7BrptATTWzZ6JFMBanRHbdwgjAb-CcJfJbUuhmZnJAqXQeBFSiNgNoq6PRSncL18Kt7wb3N-swJZIoVFVKUTsBg3MjfBjiEaBH0HqHufTX6WmvqrGYp7g9GHGPwrGYQ--hUFvunb2tDv9oGnPdyDis6Pa7MmhPz8UcZvn3TR95gsxPoEQ7OHv8ZrVQ2llhuIDnwThgHg2B793-zuW-esT2-XJKB2IIHRtz_3bFhHM3zY99xB64F7P1Wd-aC5B82MzGZofHUIUc1lcoK6oEIt8GBIQ00smpOYQUrdCKFVBGpZQ2TVUOCEdSUiTroYMgRBDdFStUh21gQullRnKa-2oSECpjlo8oeI7KpZQ2TVUfEdFEtKkqyFDIKSlCamLhJKA1MWPMoXmrOfCNqr0czfVfEwTFwtTZZPiBjLbzJSOELKGNOlqqImsIU26GmoKNWTktucnAM0_qiXANnM_SmuIrb4yP56FWBprOiuvtG4Yjk_2Vrv8VDNkLBI6K1hCZTsrhe8ldSQhCZ0VgVBzTWfllQJq4vhkb7Ur0FGRsUjorGAJle2sFN9RkYQkdFYEQq3c39v7Oc6K0Jz313RWXilrU3Cx8gHl2DXl_BFYwjdAZltCZwWZbgmdFWQNSeisiLcGhLuty6yVQgnpOYQ0YxkitkYjaglFxDC3WyXYV2pIQGVvvIv3obCEctyb8jdOGjoa6RKuYxFJaLWIiIR7rpXyWgpvqy0koLJ338W3VSyhHAunAm0VHY10CdexiCT0W0REwt3kxA96ZuQ4LmKPFm68Sm65CPbWEkISjshgiyjHySm_SzewCZfQdcEmXELbRayi_DEU6Qi1xCLaDbckfwwhAZW9CS9-dB5LSMKZGTSiKs9I6dhoJHRdRES7CZcEoX0koLI34cW3VSwhCQdn0IiqPCilY6OR0HUREeWPubC8ORexR-8GXZKui2hyrT3p8koRNbBFJOH8DLaKcqyc8v9Wr4lNuISui5BwrdrDLsXfchSKSKv2rEvxSo8EJJ_rgiUkoeuCRlR2FRVvbmIR5Vg55e-gDDGaas-6FP-LHglIPtcFS0hC1wWNqOwqKr6tYhHlWDkVaKtiNPmzLlrerIvYo3ezLskeLZhcSwhJ6Lpgi0hC1wVbRRIO0GCrKMfKKb9LN8WE72Zdkq6LcX-u0J9eMLlwHWpR9eFf7Yn_5v9o_Aa7H1Hrjp5cBZMprdPbEUTU-ml7ESj0yB7AJ-AQDl3fjbjboxYPx3B__7__AgAA__-14bUf
-
 # This query verifies stats collection for the hashJoiner, distinct and sorter.
 
 skipif bigendian

